src/devices/Ssd13xx/Commands/Ssd1327Commands/SetPreChargeVoltage.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Ssd13xx.Commands.Ssd1327Commands
+{
+    public class SetPreChargeVoltage : ISsd1327Command
+    {
+        /// <summary>
+        /// This command sets the first pre-charge voltage (phase 2) level of segment pins.
+        /// </summary>
+        /// <param name="level">Pre-charge voltage level.</param>
+        public SetPreChargeVoltage(byte level = 0x05)
```

Comment:
unit missing

Reply:
There's no units on this value but rather mapping between this value and real voltage. So I added short info from datasheet.